### ISSUE TITLE
Battery conservation mode: IO error

### ISSUE THREAD

**Issue by sssemil** [jrandiny/yoga-slim7-ubuntu#35]
After updating to 5.12.1, the following error occurs:
```
echo 0 > /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode
bash: echo: write error: Input/output error
```

Can still be disabled/enabled from Windows 10 with the Lenovo Vantage app.

**Comment by Creatik** [comment]
The same error was on this core rolled back

**Comment by sssemil** [comment]
Update: https://bugzilla.kernel.org/show_bug.cgi?id=213109
TLDR: Will be patched in 5.13
